config/babel/.babelrc

```diff
@@ -1,4 +1,4 @@
 {
-  "presets": ["es2015", "stage-1", "react"],
+  "presets": ["react", "react-native-stage-0"],
```

Comment:
Does this now make all of victory stage 0? Seems like that's not the best way to do it.

I think we may wish to have _two_ different babelrc's -- one for normal victory, one for native. (Unless for some reason they can't be extricated)